newswires/client/src/WireItemList.tsx

```diff
@@ -150,14 +174,17 @@ const WirePreviewCard = ({
 
 	const mainHeadingContent = decideMainHeadingContent(content);
 
-	const maybeSecondaryCardContent = decideSecondaryCardContent(content);
+	const maybeSecondaryCardContent = SecondaryCardContent({
```

Comment:
since this now uses a hook, it's a fully fledged component so should be called with jsx (it's probably the same either way, but components can behave oddly when called as functions)

Reply:
agree with Andrew, but in my view we should consider calling the component MaybeSecondaryCardContent or SecondaryCardContentIfApplicable, move the <p> into its definition and then instantiate it inline in the final return block of `WirePreviewCard` like... ```<MaybeSecondaryCardContent {...content} highlight={highlight} />```

Reply:
Thanks both! I've done as suggested 👍